src/guiFormSpecMenu.cpp

```diff
@@ -746,7 +743,8 @@ void GUIFormSpecMenu::parseTable(parserData* data,std::string element)
 		spec.ftype = f_Table;
 
 		for (unsigned int i = 0; i < items.size(); ++i) {
-			items[i] = unescape_string(items[i]);
+			items[i] = unescape_string(wide_to_utf8(
+				remove_escapes(utf8_to_wide(items[i]))));
```

Comment:
Yes, on windows there are [surrogate pairs](https://en.wikipedia.org/wiki/Universal_Character_Set_characters#Surrogates). I'm not sure whether linux has them too, or uses the code points.